sdk/nodejs/yarn.lock

```diff
@@ -45,9 +45,9 @@
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/@protobufjs/utf8/-/utf8-1.1.0.tgz#a777360b5b39a1a2e5106f8e858f2fd2d060c570"
 
-"@pulumi/pulumi@^0.15.1":
-  version "0.15.1"
-  resolved "https://registry.yarnpkg.com/@pulumi/pulumi/-/pulumi-0.15.1.tgz#1b94077ec0bfb8fef868d6aa66643d7f726c0678"
+"@pulumi/pulumi@dev":
+  version "0.16.1-dev-1539638908-g00a22b27"
```

Comment:
If we want to release this package today - we would ideally not depend on unreleased `@pulumi/pulumi` builds.  Can we work around the issue in that layer for now so that we can release a version of the Kubernetes package based on 0.16 of the `@pulumi/pulumi` package?  Or is a patch release of `@pulumi/pulumi` strictly necessary to deliver this fix in `@pulumi/kubernetes`?

Reply:
Yep. Let's do it.